### PR TITLE
Make airflow package type check compatible

### PR DESCRIPTION
**Pull request by houqp**
This is needed in order to type check DAGs in CI/CD pipeline

see: https://www.python.org/dev/peps/pep-0561/

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Review by potiuk — COMMENTED** [review]
Nice!

**Review by potiuk — APPROVED** [review]

**Comment by houqp** [comment]
@potiuk looks like the setup.py change is causing CI failures, before I dig into this, just wanted to check with you to see if this error looks familiar to you:

```
Fixing ownership of mounted files
Fixed ownership of mounted files

ERROR! Setup.py changed since last time requirements were generated

     When you update setup.py, you have to run

           breeze generate-requirements --python 3.6


###########################################################################################
                   EXITING /opt/airflow/scripts/ci/in_container/run_generate_requirements.sh WITH STATUS CODE 1
###########################################################################################
Fixing ownership of mounted files
Fixed ownership of mounted files
###########################################################################################
                   EXITING WITH STATUS CODE 1
###########################################################################################

Finished the script ci_generate_requirements.sh
Elapsed time spent in the script: 1 seconds
Exit code 1
```

**Comment by potiuk** [comment]
Yep. You need to do exactly what the message says :) 


>      When you update setup.py, you have to run
> 
>            breeze generate-requirements --python 3.6


See https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pinned-requirement-files for exact explanation why you need to do it :)

File: scripts/ci/in_container/run_generate_requirements.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ if [[ ${STORED_SETUP_PY_HASH} != "${CURRENT_SETUP_PY_HASH}" && ${CHECK_REQUIREME
     echo
     echo "ERROR! Setup.py changed since last time requirements were generated"
     echo
-    echo "     When you update setup.py, you have to run"
+    echo "     When you update setup.py, you have to run the following command and commit the change:"
```
Comment (houqp) [inline scripts/ci/in_container/run_generate_requirements.sh:35]: @potiuk minor change to the wording to make it more explicit :)

**Comment by houqp** [comment]
Thanks @potiuk for the quick reply, updated all pinned versions.

**Review by potiuk — APPROVED** [review]

**Pull request merged**